spot_driver_cpp/include/spot_driver_cpp/interfaces/rclcpp_wall_timer_interface.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#pragma once
+
+#include <rclcpp/node.hpp>
+#include <rclcpp/timer.hpp>
+#include <spot_driver_cpp/interfaces/timer_interface_base.hpp>
+
+#include <chrono>
+#include <memory>
+
+namespace spot_ros2
+{
+/**
+ * @brief Implements TimerInterfaceBase to use rclcpp's WallTimer.
```

Comment:
```suggestion
 * @brief Implements Spot's TimerInterfaceBase to use rclcpp's WallTimer.
```

Reply:
Strictly speaking we're implementing our own `TimerInterfaceBase` class here rather than a part of the Spot SDK, so I think it's more correct to keep this comment written as-is.